apis/python/src/tiledbsoma/utils.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
+import pandas as pd
+from tiledbsoma import pytiledbsoma as clib
+from typing import Any
+import tiledb
+from .options import SOMATileDBContext
+from .options._soma_tiledb_context import _validate_soma_tiledb_context
+
+
+def build_index(keys: np.typing.NDArray[np.int64], context: SOMATileDBContext) -> clib.IntIndexer:
```

Comment:
Since this is kind of public-facing, it should probably include a docstring. This might be a good start:

```
"""Builds an indexer object compatible with :meth:`pd.Index.get_indexer`."""
```